### PR TITLE
$this->models is a collection : Should check via isEmpty()

### PR DESCRIPTION
**Pull request by lukepolo**

### CODE REVIEW

File: src/Jobs/MakeSearchable.php [hunk 1]
```diff
@@ -35,7 +36,7 @@ public function __construct($models)
      */
     public function handle()
     {
-        if (empty($this->models)) {
+        if (!$this->models->count()) {
```
Comment (jhoff) [inline src/Jobs/MakeSearchable.php]: `$this->models->isEmpty()` would read better IMO.

File: src/Jobs/MakeSearchable.php [hunk 2]
```diff
@@ -13,17 +14,17 @@ class MakeSearchable implements ShouldQueue
     /**
      * The models to be made searchable.
      *
-     * @var \Illuminate\Database\Eloquent\Collection
+     * @var Collection
```
Comment (jhoff) [inline src/Jobs/MakeSearchable.php]: Docblocks should have fully qualified class names.

**Comment by lukepolo** [comment]
@jhoff thanks for opinions , updated~

**Comment by jhoff** [comment]
Your PR tile is outdated now

**Pull request merged**